src/component/qrcode-scanner.js

```diff
@@ -41,8 +42,13 @@ export default class QRCodeScanner extends React.Component {
     this.setState({ hasCameraPermission: status === 'granted' });
   }
 
+  handleCodeScanned({ type, data }) {
+    this.setState({ scanned: true });
+    this.props.onQRCodeScanned({ type, data });
```

Comment:
Seems you could just pass `data` and not `type`?

Reply:
Yeah we only use `data`. But no need to remove type from the QR component. Maybe the component will be used somewhere we need type later.